src/components/results.js

```diff
@@ -1,40 +1,159 @@
 import React, {Component} from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View, Text, StyleSheet, Dimensions, PixelRatio, FlatList } from 'react-native';
+import {responsiveFontSize} from 'react-native-responsive-dimensions';
+import {ListItem, Avatar} from 'react-native-elements';
 import {PropTypes} from 'prop-types';
 
-class ResultSuccess extends Component {
+/**
+ * @overview
+ * One of ResultSuccess or ResultFailure is displayed
+ * The success/failure views are children of ResultHandler
+ * ResultHandler is a child of ResultView, the parent component
+ */
+
+ /**
+  * @temp
+  * the top three results are being hard-coded for now.
+  * Eventually they will be retreived from the server
+  */
+
+var topThree = [
+    {
+      id: "randomstring1",
+      product: "Toshiba AbC 123",
+      avTitle: "Ts",

```

Comment:
These titles are a nice idea, but we may have to either make the manually by selecting the first two letters of the company, or else we can find images and add them to the server. But this is a fine way of displaying for now.